### ISSUE TITLE
LaTeX Class for OpenDreamKit Reports

### ISSUE THREAD

**Issue by kohlhase** [OpenDreamKit/OpenDreamKit#168]
We should have a LaTeX class for the project reports that has the logos, etc. I saw that Benoit used amsbook for the data plan (which is not very adequate). Having a general class ```odkreport.cls``` would help standardize appearance. @nthiery

**Comment by bpilorget** [comment]
Yep, actually Nicolas showed me how to use lateX a week ago, so I basically just copied/paste the "class" he showed me.
Having a standardised class for all ODK documents would be great, and it was on my todo list. 

Would you be able to create one, assuming you are given ideas and directions? It would be good to have finally chosen our logo beforehand. @nthiery said he would do a poll soon

**Comment by kohlhase** [comment]
I can certainly help.

**Comment by nthiery** [comment]
By the way: I have added a preliminary class for our deliverable reports. It reuses the data from the proposal to make those ultra short when there is not much more to say than what's already on the corresponding issue. See e.g. [2]
[1] https://github.com/OpenDreamKit/OpenDreamKit/blob/master/Proposal/LaTeX-proposal/deliverablereport.cls
[2] https://github.com/OpenDreamKit/OpenDreamKit/blob/master/WP1/D1.1/report.tex

When implementing this, I pondered whether latex knew how to read/write simple yaml files. This would indeed be a nice format for the pdata files, as this would make it easy to share data with other applications like our web page.

**Comment by kohlhase** [comment]
I tried the class, but ```../../media/logos/Flag_of_Europe.png``` seems to be missing did you commit? 

Also I find the dependence on final.pdata a bit overkill, 
* this file has to be made first (but we could commit it where it is; we probably should) 
* it will not change over the time of the project, so we could just inline it where it is.

**Comment by kohlhase** [comment]
> When implementing this, I pondered whether latex knew how to read/write simple yaml files. This would indeed be a nice format for the pdata files, as this would make it easy to share data with other applications like our web page.

that is a good idea. If you give an example of what you want to see, I can probably write it.

**Comment by nthiery** [comment]
Oops, I just pushed the missing logos.

I agree, we should commit the pdata file. And possibly inline it once for all. Although I did find typos while writing the reports, and fixed and rebuilt the proposal and pdata accordingly. So there can be little changes occasionally. But we could also just leave the proposal as it is, and just fix the pdata.

For some preliminary examples, we have the following files:

    https://github.com/OpenDreamKit/OpenDreamKit.github.io/tree/master/_data

which duplicate part of the pdata. So I would envision having all the pdata being written in the same format. Let me know if it would be helpful to provide a complete file containing examples for the different kind of data we have in the pdata.

Cheers,
